src/worldcereal/openeo/inference.py

```diff
@@ -120,22 +130,26 @@ def predict(self, features: np.ndarray) -> np.ndarray:
         # Prepare input data for ONNX model
         outputs = self.onnx_session.run(None, {"features": features})
 
-        # Get info on classes from the model
-        class_params = eval(
-            self.onnx_session.get_modelmeta().custom_metadata_map["class_params"]
-        )
-
         # Get classes LUT
-        LUT = dict(zip(class_params["class_names"], class_params["class_to_label"]))
+        lookup_table = self._parameters["lookup_table"]
```

Comment:
what's the behavior if this key is not in the parameters?

Reply:
I added a check at the beginning of the UDF